plugins/dagsvisualizer/frontend/src/index.tsx

```diff
@@ -5,6 +5,7 @@ import UTXOStore from 'stores/UTXOStore';
 import BranchStore from 'stores/BranchStore';
 import GlobalStore from 'stores/GlobalStore';
 import {Root} from 'components/Root';
+import React from 'react';
```

Comment:
This package is not used.

Reply:
It's to fix TS error: `TS2686: 'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.`. 
Apparently some packages needs it.